docs/installationguide.md

```diff
@@ -239,6 +240,7 @@ The ones relating specific Ultralight 2.0 bindings are described in the followin
 | IOTA_MQTT_PROTOCOL            | mqtt.protocol           |
 | IOTA_MQTT_HOST                | mqtt.host               |
 | IOTA_MQTT_PORT                | mqtt.port               |
+| IOTA_MQTT_VHOST               | mqtt.vhost              |
```

Comment:
As far as I understand, the new vhost parameter is for AMQP configuration, not MQTT (looking to "twin" PR https://github.com/telefonicaid/iotagent-json/pull/527 it only appears in the AMQP section of the configuration).

Is this a typo? Or maybe I'm missing something? :)

Reply:
Hello @fgalan and sorry for the delay. Yes, it was a typo. I've updated the PR. Thanks.